app/src/utils/utils.ts

```diff
@@ -28,22 +42,69 @@ export function isValidAddress(val: string | undefined) {
   return val && isAddress(val);
 }
 
-// Expects a unix timestamp and will return a human readable message of how far in the past/future it is
-export function daysAgo(val = 0) {
-  // Use a formatter to establish "in 10 days" vs "10 days ago"
-  const formatter = new Intl.RelativeTimeFormat();
-  // Number of days since now
-  const deltaDays = (val * 1000 - Date.now()) / (1000 * 3600 * 24);
+// --- Grants + Cart ---
+const CART_KEY = 'cart';
```

Comment:
do we wanna wrap this within a dgrants-core object ? cause cart is a generic name so by wrapping it within a dgrants object, we avoid random conflicts with other things (like imagine gitcoin cgrants vs dgrants) 
Or the other option is we use a prefix `dgrant_` for any localStorage variable we create

Reply:
What exactly do you mean by a "dgrants-core object"? Local storage is domain-specific, so the naming can only conflict with local storage keys within the dgrants app, and never with cgrants, so by definition (almost) anything in localstorage must be dgrants specific. 

The only exception is that browser extensions can read/write from local storage, though I'm not worried about a conflict there.

If we do want to prefix local storage keys, we also have to update the key names defined in `app/src/store/settings.ts`, and therefore it's probably worth it's own PR

Reply:
Ah yup you're right ! ignore my comment